schema.js

```diff
@@ -1,34 +1,28 @@
 const axios = require('axios');
-const { GraphQLObjectType, GraphQLFloat, GraphQLList, GraphQLSchema, GraphQLString, GraphQLScalarType, GraphQLInt} = require('graphql');
+const { GraphQLObjectType, GraphQLFloat, GraphQLSchema, GraphQLString} = require('graphql');
 
 // Bitcoin data type
-const rateData = new GraphQLObjectType ({
-  name: 'Rates',
+const CurrencyRateType = new GraphQLObjectType({
+  name: 'CurrencyRateType',
   fields: () => ({
-    name: {type: GraphQLString},
-    rate: {type: GraphQLString}
-  })
-})
-const LaunchType = new GraphQLObjectType({
-  name: 'launch',
-  fields: () => ({
-    AED: {type: rateData},
-    AFN: {type: rateData},
-    AMD: {type: rateData},
-    ANG: {type: rateData},
-    AOA: {type: rateData}
+    code: {type: GraphQLString},
+    symbol: {type: GraphQLString},
+    rate_float: {type: GraphQLFloat}
   })
 });
 
 //Root Query
-
 const RootQuery = new GraphQLObjectType({
   name: 'RootQueryType',
   fields: {
-    launchInfo: {
-      type: new GraphQLList(LaunchType),
-      resolve(parent, args) {
-        return axios.get('https://apiv2.bitcoinaverage.com/constants/exchangerates').then(res => res.data);
+    bitcoinRates: {
+      type: CurrencyRateType,
+      args: {
+        currency: {type: GraphQLString}
+      },
+      resolve: async(_, args) => {
+        const coinData = await axios.get('https://api.coindesk.com/v1/bpi/currentprice.json').then(res => res.data.bpi);
+        return coinData[args.currency];
```

Comment:
Here is the API request i'm making. Here's an output of the GET request for reference:

<img width="1440" alt="Screen Shot 2020-07-05 at 6 50 43 PM" src="https://user-images.githubusercontent.com/46545681/86548233-7ace7280-bef0-11ea-9e92-046e3f56a223.png">

We filter this data to our type later.